docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+services:
```

Comment:
the docker compose is not mentioned in the readme. how does it play together with the summarizer? where does the model "live"? is it just pulled up together with the app?